run-order.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ $# -ne 2 ]]; then
+  echo "Invalid arguments. Use: $0 SRC DST"
+  exit 1
+fi
+
+SRC_REPOSITORY="$1"
+ORDERED_REPOSITORY="$2"
+shift 2
+
+if [[ ! -d "${SRC_REPOSITORY}" ]]; then
+  echo "Invalid argument. ${SRC_REPOSITORY} has to be a directory."
+  exit 1
+fi
+
+if [[ -e "${ORDERED_REPOSITORY}" ]]; then
+  echo "Invalid argument. ${ORDERED_REPOSITORY} may not exist."
+  exit 1
+fi
+
+
+BASE="$(dirname "$(readlink -f "$0")")"
+SETUP_MERCURIAL="${BASE}/setup-mercurial.sh"
+VIRTUALENV="${BASE}/data/py3-env"
+
+if ! /bin/bash "${SETUP_MERCURIAL}"; then
+  echo "Error during setup."
+  exit 2
+fi
+source "${VIRTUALENV}/bin/activate"
+
+# Disable all extensions.
+# (https://stackoverflow.com/questions/46612210/mercurial-disable-all-the-extensions-from-the-command-line)
+HGRCPATH= hg clone "http://hg.fast-downward.org" "${ORDERED_REPOSITORY}"
+set +e  # hg incoming has an non-zero exit code if nothing is incoming
+CHANGESETS="$(hg incoming -R "${SRC_REPOSITORY}" --template "{node} " --quiet "${ORDERED_REPOSITORY}")"
+set -e
+if [[ ! -z "${CHANGESETS}" ]]; then
+  HGRCPATH= hg --config extensions.strip= strip ${CHANGESETS} --nobackup -R "${ORDERED_REPOSITORY}"
```

Comment:
One quick comment: I'd move the `-R` to the front in all places (directly after `hg` if this is possible).

Reply:
And while we're at it: maybe use HGPLAIN= **in addition** to  HGRCPATH= (see second answer at https://stackoverflow.com/questions/52584763/invoke-mercurial-without-loading-global-and-user-configs, this should ensure that we always get the same output from hg.)

Reply:
I'd also add a newline with a line continuation (`\`) after `HGRCPATH= ` because it makes it easier to read. Of course this applies to the other places where this is used as well. Together with Jendrik's comment, I'm suggesting
```
HGRCPATH= HGPLAIN= \
hg -R ${REPO} command options
```
or, if the options get very long
```
HGRCPATH= HGPLAIN= \
hg -R ${REPO} command \
    option1 \
    option2 \
    option3
```

Reply:
in some comment Florian suggested to use export HGRC=/export HGPLAIN. This is also now done.